ardent/distribution.yaml

```diff
@@ -275,11 +275,11 @@ repositories:
       tags:
         release: release/ardent/{package}/{version}
       url: https://github.com/ros2-gbp/fastrtps-release.git
-      version: 1.5.0-10
+      version: 1.5.0-11
     source:
       type: git
       url: https://github.com/eProsima/Fast-RTPS.git
-      version: master
+      version: bfb448e19bff253293e8f28ee8c3d3d78d882be2
```

Comment:
Other packages use the `master` branch. Should we keep the same here?

Reply:
I think this was probably overzealous and leaving the master branch seems like the sensible analogue.